koopovereenkomst-v3-simple/pages/index.tsx

```diff
@@ -1,42 +1,100 @@
+import { Box, Container, Typography, useMediaQuery } from "@mui/material";
 import Head from "next/head";
-import Link from "next/link";
-import Layout, { siteTitle } from "../components/layout";
-import styles from "../styles/index.module.css";
-import utilStyles from "../styles/utils.module.css";
+import Layout, { siteTitle } from "../src/Layout";
+import Link from "../src/Link";
 
 export default function Home() {
+
+  const minimalWidth = useMediaQuery('(max-width: 600px)');
+
+  const cardStyle = {
```

Comment:
Zo staat de CSS wel dichter bij de code ... maar of het er nu overzichtelijker en beter van wordt? 🤔 😏

Reply:
Hier woedt ook wel een felle strijd over hoor tussen voor- en tegenstanders. Kan zo niet uit m'n hoofd zeggen wat precies de voor- en tegenargumenten van beide kampen zijn, voor nu het best als we proberen consistent te blijven binnen ons project 😄 Enne,... dit is ook wat de template hanteert waar wij op voort bouwen